utils/polus-bfio-util/bfio/backends.py

```diff
@@ -0,0 +1,787 @@
+from tifffile import tifffile
+from pathlib import Path
+import numpy
+from concurrent.futures import ThreadPoolExecutor
+import bfio
+from bfio.OmeXml import OMEXML
+import bfio.base_classes
+import struct, copy, zlib, io, typing, logging, threading
+
+logging.basicConfig(format='%(asctime)s - %(name)-8s - %(levelname)-8s - %(message)s',
+                    datefmt='%d-%b-%y %H:%M:%S')
+logger = logging.getLogger("bfio.backends")
+
+class PythonReader(bfio.base_classes.AbstractReader):
+    
+    logger = logging.getLogger("bfio.backends.PythonReader")
+    
+    _rdr = None
+
+    def __init__(self, frontend):
+        super().__init__(frontend)
+
+        self.logger.debug('__init__(): Initializing _rdr (tifffile.TiffFile)...')
```

Comment:
Double quote for string?

Reply:
Python does not enforce strict typing on Strings, so single or double is okay. However, I should be more consistent in my formatting, so I will change it.